portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,39 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  ArrayList<String> comments = new ArrayList<String>();
```

Comment:
nit: `List<String> comments = new ArrayList<>();` offers more [flexibility & information hiding](http://jtechies.blogspot.com/2012/07/item-52-refer-to-objects-by-their.html)